resources/landmarks/landmarks.json

```diff
@@ -0,0 +1,390 @@
+{
+  "independence": {
+    "name": "Independence, Missouri",
+    "id": 0,
+    "isCity": true,
+    "isRiver": false,
+    "altRoute": false,
+    "sayings": [
+      "first saying",
+      "second saying",
+      "third saying"
+    ],
+    "nextPoints": [
+      {
+        "id": 1,
+        "name": "Kansas River Crossing",
```

Comment:
The name shouldn't be duplicated. You should be able to find the name from the id during runtime.

Reply:
I thought about this a lot before I did it, and I honestly think its simpler to just have the name variable twice.  When I'm instantiating the point array, the next node isn't yet available for me to grab the name value from.  Which results in storing a null value.  Then, after the list is completely populated I would have to loop through it once for each destination to match the ID and pull the name.  By storing the name inside the JSON for each destination it improves the readability of the JSON and allows us to store the name in the point array immediately and avoid using null values.